File: OPENAPI-REST-API/swagger-client/dart/lib/model/ticket_new.dart
part of swagger.api;

class TicketNew {
  
  String subject = null;

  String body = null;

  int serviceId = null;

  String serviceModule = null;

  TicketNew();

  @override
  String toString() {
    return 'TicketNew[subject=$subject, body=$body, serviceId=$serviceId, serviceModule=$serviceModule, ]';
  }

  TicketNew.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    subject = json['subject'];
    body = json['body'];
    serviceId = json['service_id'];
    serviceModule = json['service_module'];
  }

  Map<String, dynamic> toJson() {
    return {
      'subject': subject,
      'body': body,
      'service_id': serviceId,
      'service_module': serviceModule
     };
  }

  static List<TicketNew> listFromJson(List<dynamic> json) {
    return json == null ? new List<TicketNew>() : json.map((value) => new TicketNew.fromJson(value)).toList();
  }

  static Map<String, TicketNew> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, TicketNew>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new TicketNew.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/TicketNew.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// New helpdesk ticket request.  
  /// </summary>
  [DataContract]
  public class TicketNew {
    /// <summary>
    /// Gets or Sets Subject
    /// </summary>
    [DataMember(Name="subject", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "subject")]
    public string Subject { get; set; }

    /// <summary>
    /// Gets or Sets Body
    /// </summary>
    [DataMember(Name="body", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "body")]
    public string Body { get; set; }

    /// <summary>
    /// Gets or Sets ServiceId
    /// </summary>
    [DataMember(Name="service_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "service_id")]
    public int? ServiceId { get; set; }

    /// <summary>
    /// Gets or Sets ServiceModule
    /// </summary>
    [DataMember(Name="service_module", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "service_module")]
    public string ServiceModule { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TicketNew {\n");
      sb.Append("  Subject: ").Append(Subject).Append("\n");
      sb.Append("  Body: ").Append(Body).Append("\n");
      sb.Append("  ServiceId: ").Append(ServiceId).Append("\n");
      sb.Append("  ServiceModule: ").Append(ServiceModule).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
